.github/actions/build_debug/action.yml

```diff
@@ -18,6 +18,8 @@ runs:
     - uses: actions-rs/toolchain@v1
       with:
         target: ${{ matrix.config.target }}
+        toolchain: nightly
```

Comment:
We have defined the nightly in the rust-toolchain, why here still need? :thinking:

Reply:
emm, rust-toolchain.toml hints at the format of the file, which I think is better. Unfortunately, action-rs/toolchain does not seem to support direct processing and will prompt that the toolchain file cannot be found.